backend/main.py

```diff
@@ -613,10 +611,7 @@ async def dispatch(self, request: Request, call_next):
             try:
                 body, model, user = await get_body_and_model_and_user(request)
             except Exception as e:
-                return JSONResponse(
-                    status_code=status.HTTP_400_BAD_REQUEST,
-                    content={"detail": str(e)},
-                )
+                return exception_json_response(e)
```

Comment:
Not a huge fan of abstracting away `exception_json_response`, let's use `JSONResponse` directly here.